src/CBT.Core/Tasks/WriteItemToJsonFile.cs

```diff
@@ -38,6 +38,12 @@ public bool Execute()
             }
             PackageRestoreData assetsFile = new PackageRestoreData
             {
+                ProjectJsonPath = Input
```

Comment:
There's no null checking here, are we guaranteed to get an item for each of these values?